Week3/homework/js-exercises/Project-2-JS1/creditCardValid.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+function cardValidator(cardNum) {
+    let result = "";
+    const digits = cardNum.split("");
+    const lastDigit = digits[digits.length - 1];
+    const sumOfDigits = digits.reduce((total, item) => {
+        return Number(total) + Number(item)
+    });
+    if (cardNum.toString().length !== 16 || Number.isInteger(Number(cardNum)) === false) {
```

Comment:
A nice idea would be to move all the `if-else-if` logic to separate functions. Code would be much nicer and reusable then.